lib/ansible/config/base.yml

```diff
@@ -215,6 +215,14 @@ CACHE_PLUGIN_TIMEOUT:
   - {key: fact_caching_timeout, section: defaults}
   type: integer
   yaml: {key: facts.cache.timeout}
+COLLECTION_PATHS:
+  name: ordered list of root paths for loading installed Ansible collections content
+  default: ['~/.ansible/collections', '/usr/share/ansible/collections']
+  type: list
+  env:
+  - {name: ANSIBLE_COLLECTION_PATHS}
+  ini:
+  - {key: collection_paths, section: defaults}
```

Comment:
use `type: pathspec`, the `:` separator is safe, while `,` is valid in paths, that is why `pathspec` exists.

Reply:
done